homeassistant/components/vizio/config_flow.py

```diff
@@ -152,7 +153,17 @@ async def async_step_init(
                         default=self.config_entry.options.get(CONF_APPS, {}).get(
                             default_include_or_exclude, []
                         ),
-                    ): cv.multi_select(VizioAsync.get_apps_list()),
+                    ): cv.multi_select(
+                        [
+                            APP_HOME["name"],
+                            *sorted(
```

Comment:
Would it make sense to just store these apps sorted anyway ?

Reply:
good idea 👍